bin/cli.js

```diff
@@ -93,6 +99,14 @@ const argv = yargs
       alias: 'v'
     }
   })
+  .command('get <target>', 'get latest documented version or reference', (yargsGet) => {
+    yargsGet.positional('target', {
+      alias: 't',
```

Comment:
Does this mean that I could also do `versionist -t reference` ?

Reply:
Good catch, options declared inside a command apply only for the command, so anything in there is only valid when you do `versionist get ...`. Having said this, I could not find way to use the alias in a sub-command so I removed it and added `choices` instead